package.json

```diff
@@ -182,6 +183,7 @@
     "jsonwebtoken": "9.0.2",
     "metro-react-native-babel-preset": "0.77.0",
     "patch-package": "6.4.7",
+    "pretty": "2.0.0",
```

Comment:
Is this really necessary? "2.0.0 • Public • Published 8 years ago" seems unmaintained and it only makes HTML more readable?

Reply:
Ah, it's a dev dependency. Sorry, nevermind!